website/common/script/content/loginIncentives.js

```diff
@@ -243,7 +242,7 @@ module.exports = function getLoginIncentives (api) {
   });
 
   let prevRewardKey;
-  _.range(MAX_INCENTIVES).forEach(function addPrevRewardLink (index) {
+  _.range(MAX_INCENTIVES + 1).forEach(function addPrevRewardLink (index) {
```

Comment:
Looks good, although I am weary about this +1 now, but I don't think it should hold us up.